### PR TITLE
Capitalize examples where appropriate.

### PR DESCRIPTION
**Pull request by Eelis**
For consistency.

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -5414,7 +5414,7 @@
 \pnum
 Implementations should avoid the use of dynamically allocated memory for a small contained object.
 \begin{example}
-where the object constructed is holding only an \tcode{int}.
+Where the object constructed is holding only an \tcode{int}.
```
Comment (jensmaurer) [inline source/utilities.tex]: Could you please leave this untouched in your change and open an edit issue for rephrasing this?

**Comment by jensmaurer** [comment]
Otherwise, looks good to me.

File: source/special.tex [hunk 2]
```diff
@@ -33,7 +33,7 @@
 \pnum
 Programs may explicitly refer to implicitly-declared special member functions.
 \begin{example}
-a program may explicitly call, take the address of or form a pointer to member
+A program may explicitly call, take the address of or form a pointer to member
```
Comment (jwakely) [inline source/special.tex]: There should be a comma before "or" but I'll fix that separately.

File: source/utilities.tex (hunk 1 above)
Comment (Eelis) [inline source/utilities.tex]: Done (#1360)

**Comment by tkoeppe** [comment]
@Eelis: Rebase?

**Comment by Eelis** [comment]
Rebased

**Comment by zygoloid** [comment]
Thanks, I've also added a matching rule to the style guide.

**Pull request merged**